crates/curp/tests/it/server.rs

```diff
@@ -297,8 +297,7 @@ async fn shutdown_rpc_should_shutdown_the_cluster() {
     ));
 
     let collection = collection_task.await.unwrap();
-    sleep_secs(1).await; // wait for the cluster to shutdown
-    assert!(group.is_finished());
+    group.wait_for_group_shutdown(Duration::from_secs(7)).await;
```

Comment:
As all the timeout is the same, shall we provide a default value for it or just wrap the value in the default function?

Reply:
Done